industrial_robot_client/src/joint_trajectory_action.cpp

```diff
@@ -38,44 +38,31 @@ namespace industrial_robot_client
 {
 namespace joint_trajectory_action
 {
-
 const double JointTrajectoryAction::WATCHDOG_PERIOD_ = 1.0;
 const double JointTrajectoryAction::DEFAULT_GOAL_THRESHOLD_ = 0.01;
 
 JointTrajectoryAction::JointTrajectoryAction() :
-    action_server_(node_, "joint_trajectory_action", boost::bind(&JointTrajectoryAction::goalCB, this, _1),
-                   boost::bind(&JointTrajectoryAction::cancelCB, this, _1), false), has_active_goal_(false),
-                       controller_alive_(false)
-{
-  ros::NodeHandle pn("~");
-
-  pn.param("constraints/goal_threshold", goal_threshold_, DEFAULT_GOAL_THRESHOLD_);
-
-  if (!industrial_utils::param::getJointNames("controller_joint_names", "robot_description", joint_names_))
-    ROS_ERROR("Failed to initialize joint_names.");
-
-  // The controller joint names parameter includes empty joint names for those joints not supported
-  // by the controller.  These are removed since the trajectory action should ignore these.
-  std::remove(joint_names_.begin(), joint_names_.end(), std::string());
-  ROS_INFO_STREAM("Filtered joint names to " << joint_names_.size() << " joints");
-
-  pub_trajectory_command_ = node_.advertise<trajectory_msgs::JointTrajectory>("joint_path_command", 1);
-  sub_trajectory_state_ = node_.subscribe("feedback_states", 1, &JointTrajectoryAction::controllerStateCB, this);
-  sub_robot_status_ = node_.subscribe("robot_status", 1, &JointTrajectoryAction::robotStatusCB, this);
-
-  watchdog_timer_ = node_.createTimer(ros::Duration(WATCHDOG_PERIOD_), &JointTrajectoryAction::watchdog, this, true);
-  action_server_.start();
-}
-
-JointTrajectoryAction::~JointTrajectoryAction()
-{
-}
-
+    action_server_(node_, "joint_trajectory_action", boost::bind(&JointTrajectoryAction::goalCB, this, _1), boost::bind(&JointTrajectoryAction::cancelCB, this, _1), false), has_active_goal_(false), controller_alive_(false) {
```

Comment:
can you go back to the other style, as the other style was more correct.

 - Avoid long lines (in our repos we said max 100 characters per line)
 - Fix the indentation again (below)